packages/TorneloScoresheet/src/style/colour.ts

```diff
@@ -50,6 +55,8 @@ export const textColour = (colour: ColourType): string => {
     case colours.negative:
     case colours.darkenedElements:
     case colours.black:
+    case colours.lightBlue:
+    case colours.darkBlue:
```

Comment:
Out of scope for this PR, but we may need to change the names here from `colours` to `colors` to keep consistency with CSS and and most other functions.

Reply:
yeh maybe well have to have  a chat about it

Reply:
Yeah the approach so far has been to consistently use `colour` for anything we write. Obviously we can't control the terminology React Native uses (`color`). I think it's fine that we use `colour`, because it's immediately obvious that it's our code, as long as we are consistent. I don't really care either way tho